exercises/say/example.rs

```diff
@@ -0,0 +1,66 @@
+const SMALL: &'static [&'static str] = &["zero", "one", "two", "three", "four", "five", 
+                                         "six", "seven", "eight", "nine", "ten", 
+                                         "eleven", "twelve", "thirteen", "fourteen", "fifteen", 
+                                         "sixteen", "seventeen", "eighteen", "nineteen"];
+
+const TENS: &'static [&'static str] = &["ones", "ten", "twenty", "thirty", "forty",
+	                                    "fifty", "sixty", "seventy", "eighty", "ninety"];
+
+const SCALE: &'static [&'static str] = &["", "thousand", "million", "billion",
+	                                     "trillion", "quadrillion", "quintillion"];
+
+pub fn encode(n: i64) -> Result<String, String> {
+    if n < 0 {
+        Err(String::from("Input cannot be negative."))
+    } else if n > 999_999_999_999 {
+        Err(String::from("Max input is 999,999,999,999"))
+    } else {
+        Ok(encode_u64(n as u64))
+    }
+}
+
+pub fn encode_u64(n: u64) -> String {
+    if n < 20 {
+        SMALL[n as usize].to_string()
+    } else if n < 100 {
+        let small = n % 10;
+        let mut out = String::from(TENS[n as usize / 10]);
+        if small > 0 {
+            out.push('-');
+            out.push_str(SMALL[small as usize]);
+        }
+        out
+    } else if n < 1000 {
+		let mut out = String::from(SMALL[n as usize / 100]);
```

Comment:
indentation seems unusual on these lines, as if spaces and tabs were mixed. can you check?